src/Sylius/Bundle/ProductBundle/Form/EventSubscriber/ProductOptionFieldSubscriber.php

```diff
@@ -46,7 +46,7 @@ public function preSetData(FormEvent $event): void
         /** Options should be disabled for configurable product if it has at least one defined variant */
         $disableOptions = (null !== $this->variantResolver->getVariant($product)) && $product->hasVariants();
 
-        $form->add('options', ProductOptionChoiceType::class, [
```

Comment:
From what I see, the `ProductOptionChoiceType` is no longer used. As we can't remove it now (BC policy) let's deprecate it at least.

Reply:
The deprecation will go in a separate PR targeting 1.13